parkhangs-webapp/src/features/modal/ModalMapDetail.js

```diff
@@ -59,12 +120,15 @@ class ModalMapDetail extends React.Component {
 
 const mapStateToProps = (state) => {
     return {
-        parks: state.parks.parks
+        parks: state.parks.parks,
+        eventsById: state.parks.eventsById, //universal; actions for fulfillment and rejected not connecting? array remains empty
+        error: state.parks.error
     }
 }
 
 const mapDispatchToProps = (dispatch) => ({
     deleteEventFromPark: (parkId, eventId) => dispatch(deleteEvent(parkId, eventId)),
+    fetchEvents: (parkId) => dispatch(fetchEventsById(parkId)) //should then cases be added here?
```

Comment:
I think callbacks related to these actions should be handled within the actions maybe, like this action should dispatch another action that stores the success/failure of the request in the store and can be referenced from this component.